thrust/thrust/detail/vector_base.inl

```diff
@@ -1273,7 +1273,7 @@ namespace detail
 template <typename InputIterator1, typename InputIterator2>
 bool vector_equal(InputIterator1 first1, InputIterator1 last1,
                   InputIterator2 first2,
-                  thrust::detail::true_type)
+                  thrust::detail::true_types)
```

Comment:
I did try `cuda::std::true_type` here but it failed due to an error I cannot recall. I can revisit later though when i have time.